### PR TITLE
path-extractor (new formula)

### PR DESCRIPTION
**Pull request by tomfitzhenry**

### CODE REVIEW

**Comment by tomfitzhenry** [comment]
Awaiting a stable version being released, which is being discussed in https://github.com/edi9999/path-extractor/issues/4

**Comment by edi9999** [comment]
I just tagged `v0.1.0`

**Comment by tomfitzhenry** [comment]
Thanks @edi9999.

Reviews from Homebrew maintainers would be appreciated.

File: Library/Formula/path-extractor.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = bin
+    ENV["GOPATH"] = buildpath
+    ENV["GOHOME"] = buildpath
+
+    mkdir_p buildpath/"src/github.com/edi9999/"
+    ln_sf buildpath, buildpath/"src/github.com/edi9999/path-extractor"
+    Language::Go.stage_deps resources, buildpath/"src"
```
Comment (mikemcquaid) [inline Library/Formula/path-extractor.rb]: There aren't any resources being used here. Perhaps there should be if it has any go dependencies?

File: Library/Formula/path-extractor.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    system "go", "build", "-o", bin/"path-extractor", "path-extractor/pe.go"
+  end
+
+  test do
+    assert `printf 'a\nfoo/bar/baz\nd' | #{bin}/path-extractor` == "foo/bar/baz\n"
```
Comment (mikemcquaid) [inline Library/Formula/path-extractor.rb]: Use `pipe_output` and `assert_match` instead here.

**Comment by DomT4** [comment]
Mike's two comments further up still need addressing, Thanks!

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew, @tomfitzhenry; we appreciate it! :tada:

**Pull request closed without merging**